### PR TITLE
Check unreleased

### PR DESCRIPTION
**Pull request by alex-e-leon**
Adds an optional flag to check whether the unreleased section has content - can be used in CI to make sure that the changelog has been updated before a merge into master.

@jesstelford I also removed the scripts directory - since I'm not too sure what it was used for - maybe just for initial development?

### CODE REVIEW

File: scripts/version-changelog.js [hunk 1]
```diff
@@ -1,105 +0,0 @@
-#!/usr/bin/env node
```
Comment (jesstelford) [inline scripts/version-changelog.js:1]: This file was later published as the package [`version-changelog`](https://github.com/jesstelford/version-changelog), so is no longer needed here 👍

**Review by jesstelford — APPROVED** [review]
Added a few little tweaks, but otherwise looks great! 👍 
Thanks!

**Comment by jesstelford** [comment]
🤣 This PR didn't contain any changes to the `CHANGELOG.md`

**Comment by alex-e-leon** [comment]
If only there was some kind of software that could validate if the changelog had been updated :stuck_out_tongue: !

**Comment by jesstelford** [comment]
Dogfooding FTW: https://circleci.com/gh/jesstelford/changelog-verify/4 (from PR #4)

**Pull request merged**